README.md

```diff
@@ -4,7 +4,36 @@ This is a [single-spa](https://single-spa.js.org/) React microapp that runs with
 
 > **NOTE:** This application has been configured to be run as child app of a single-spa application. So while this app can be deployed and run independently, we would need some frame [single-spa](https://single-spa.js.org/) which would load it. While technically we can achieve running this app as standalone app it's strongly not recommended by the author of the `single-spa` approch, see this [GitHub Issue](https://github.com/single-spa/single-spa/issues/640) for details.
 
->**NOTE:** To successfully run this application, you must also run the two following apps: `mfe-core` & `mfe-header`. Please see their corresponding README's for instructions on running each app. 
+>**NOTE:** To successfully run this application, you must also run the two following apps: `mfe-core` & `mfe-header`. Please see their corresponding README's for instructions on running each app.
```

Comment:
@brooketopcoder Does it makes sense to mention this confluence link(https://topcoder.atlassian.net/wiki/spaces/PROD/pages/2811920900/MFE+Setup) in the intro section? Or is this repository can be used by developers who don't have confluence access?

Reply:
@testflyjets I've now added those steps as part of this readme as well. Thanks for the suggestion :)

Reply:
@testflyjets I am working on the readme file in this repo(https://github.com/topcoder-platform/mfe-core) but I don't have access to push my changes to feature branch. Can you please provide me access to it?

cc @casey-tc